Document the required command for c2c build in MacOS (#9492)
ballerina-platform/ballerina-dev-website: swan-lake/development-tutorials/run-in-the-cloud/code-to-cloud-deployment.md

1. <a href="https://www.docker.com/" target="_blank">Docker</a> installed and configured in your machine
2. <a href="https://kubernetes.io/docs/tasks/tools/" target="_blank">Kubectl</a> installed and configured in a <a href="https://minikube.sigs.k8s.io/docs/start/" target="_blank">Kubernetes cluster</a>

> >**Note:** The macOS users with Apple silicon chip need set an environment variable `DOCKER_DEFAULT_PLATFORM` to `linux/amd64`, before building the image. This is because the Ballerina Docker image is not supported on Apple silicon chips yet.
> > ```
> > export DOCKER_DEFAULT_PLATFORM=linux/amd64
> > ```

Reviewer: ````suggestion
>**Note:** macOS users with Apple Silicon chips need to set an environment variable named `DOCKER_DEFAULT_PLATFORM` to `linux/amd64`, before building the image. This is because the Ballerina Docker image is not supported on Apple Silicon chips yet.

> ```
> export DOCKER_DEFAULT_PLATFORM=linux/amd64
> ```
````

1. <a href="https://www.docker.com/" target="_blank">Docker</a> installed and configured in your machine
2. <a href="https://kubernetes.io/docs/tasks/tools/" target="_blank">Kubectl</a> installed and configured in a <a href="https://minikube.sigs.k8s.io/docs/start/" target="_blank">Kubernetes cluster</a>

> >**Note:** For Mac users with Apple Silicon chips, run `export DOCKER_DEFAULT_PLATFORM=linux/amd64` before building the image. This is necessary because the Ballerina base image doesn't yet support the M1 chip.

Reviewer: ```suggestion
>**Note:** For Mac users with Apple Silicon chips, run `export DOCKER_DEFAULT_PLATFORM=linux/amd64` before building the image. This is necessary because the Ballerina base image doesn't support the M1 chip yet.
```

Also feels a bit inconsistent saying Apple Silicon chips and then mentioning M1.

> >**Note:** For Mac users with Apple Silicon chips, run `export DOCKER_DEFAULT_PLATFORM=linux/amd64` before building the image. This is because the Ballerina Docker image is not supported on Apple Silicon chips.

Reviewer: ```suggestion
>**Note:** For Mac users with Apple Silicon chips, run `export DOCKER_DEFAULT_PLATFORM=linux/amd64` before building the image. This is because the Ballerina Docker image is not supported on Apple Silicon chips yet.
```

> >**Note:** For Mac users with Apple Silicon chips, run `export DOCKER_DEFAULT_PLATFORM=linux/amd64` before building the image. This is because the Ballerina Docker image is not supported on Apple Silicon chips yet.

Reviewer: Instead of saying 

```
run `export DOCKER_DEFAULT_PLATFORM=linux/amd64`
```

should we say something like

> set an environment variable `DOCKER_DEFAULT_PLATFORM` to `linux/amd64`.

Author: Since this is valid only for macOS, the command should do IMO.

Reviewer: I think it'll still be good to explicitly say set an environment variable. We can give the snippet too.

Author: Fixed
